crates/valence_command_derive/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "valence_command_derive"
```

```suggestion
name = "valence_command_macros"
```

For consistency and in case we want to add more than just derives in the future.